[EXAMPLE] Add example to open a modal on mouse over on a modal (#103)
process-analytics/bpmn-visualization-examples: examples/interaction/call_activity_with_modal_on_mouse_over/README.md

# Open a modal on event on a call activity

> Javascript example to demonstrate how to navigate the BPMN diagram with the mouse.

Reviewer: ```suggestion
Javascript example to demonstrate how to open a new BPMN diagram in a modal.
```

- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/interaction/call_activity_with_modal_on_mouse_over/index.html)
- to run locally, open the [index.html](index.html) directly in a Web Browser

## ♻️ Usage

> When after the main `BpmnVisualization` instantiating, get the HTML element corresponding to the call activity to add a listener and activate the modal.

Reviewer: ```suggestion
After the main `BpmnVisualization` instantiation, get the HTML element corresponding to the call activity to add a listener and activate the modal.
```

> ⚠️ The secondary `BpmnVisualization` must be visible during the BPMN diagram loading, otherwise the elements of the graph are not visible.

Reviewer: ```suggestion
⚠️ The secondary `BPMN container` (in charge of displaying the Call Activity) must be visible during the BPMN diagram loading, otherwise, some BPMN elements won't be displayed.
```
